index.html

```diff
@@ -98,10 +98,55 @@ <h2 class="work-title">tonic</h2>
           <li>Javascript</li>
         </ul>
         <div>
-          <button type="button" class="btn">See Project</button>
+          <button data-modal-target="#modal" class="btn">See Project</button>
+
+          <div class="modal" id="modal">
+
+            <div class="modal-description">
+              <h2 class="work-title">tonic</h2>
+              <button data-btn-close class="btn-close">&times;</button>
+            </div>
+
+            <div class="modal-body">
+              <ul class="canopy">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <div class="work-img">
```

Comment:
* Kindly adjust the design of your popup on both mobile and desktop the background should be blurred as shown in the Figma design 
* ## Figma 
![image](https://github.com/Eccb7/Portfolio2.0/assets/52497246/a2901251-2a4a-4841-aec9-69d80af925fa)
![image](https://github.com/Eccb7/Portfolio2.0/assets/52497246/3191d755-5533-422a-94dc-5cac749035cb)
## Yours 
![image](https://github.com/Eccb7/Portfolio2.0/assets/52497246/fda3f7be-d7c7-40e3-94d4-bd463b1033de)